File: Iteration2-Python/src/Staff.py

from Person import Person

import logging
logger = logging.getLogger(__name__)    # to ensure that the logger is named after the module
class Staff(Person):
    def __init__(self,id,name, eMail, password, controller_instance):
        super().__init__(id, name, eMail, password, controller_instance)
        logger.info(f"Super type' 'Person' constructor is called. {__name__} type object is created.")

File: Iteration1-Java/src/main/java/cse3063group3/Staff.java
package cse3063group3;
import org.json.JSONObject;

public abstract class Staff extends Person {
    // write this contructor the uml
    public Staff (int id, String name, String eMail, String password){
        super(id, name, eMail, password); 
    }
    public  JSONObject toJSON(){return null;};
}
